ui/apps/platform/src/Components/CVEStackedPill/CVEStackedPill.js

```diff
@@ -38,12 +37,14 @@ const CVEStackedPill = ({
     url,
     fixableUrl,
     showTooltip,
-    imageNotes,
+    entityName,
     scan,
+    scanMessage,
 }) => {
     const hasCounts = vulnCounter?.all?.total > 0;
     const useScan = !!scan;
```

Comment:
`scan` is only used for these two variables. Any ideas how to improve that?

Reply:
I'd rather have the complexity in the component where it's needed, even if it's verbose, than make the caller figure it out, and pass in the two vars. ("Caller" in JSX is the parent component.)

That said, if you see a way to simplify the check overall, I'd be open to that.